monitor/install_collectd_debian.sh

```diff
@@ -50,7 +52,11 @@ sudo -H -i -u "${ROOT_USER}" systemctl enable nginx.service
 sudo -H -i -u "${ROOT_USER}" systemctl enable collectd.service
 
 echo "[*] Restarting services"
-sudo -H -i -u "${ROOT_USER}" systemctl restart bisq.service
+set +e
```

Comment:
this lets errors slip through and leave operators in the believe that everything is fine.

Something like this may do it. I am pretty sure though that if I really want to break it I still can.
```
set +e
service bisq status >/dev/null 2>&1
[ $? != 4 ] && set -e && sudo -H -i -u "${ROOT_USER}" systemctl restart bisq.service
set +e
service bisq-pricenode status >/dev/null 2>&1
[ $? != 4 ] && set -e && sudo -H -i -u "${ROOT_USER}" systemctl restart bisq-pricenode.service
set -e
```